utils/ctxutil/ctxutil_test.go

```diff
@@ -0,0 +1,66 @@
+package ctxutil
+
+import (
+	"context"
+	"testing"
+)
+
+func TestDebug(t *testing.T) {
+	if IsDebug(context.Background()) {
+		t.Errorf("Vanialla ctx should not be debug")
+	}
+	if !IsDebug(WithDebug(context.Background(), true)) {
+		t.Errorf("Should have debug flag")
+	}
+	if IsDebug(WithDebug(context.Background(), false)) {
+		t.Errorf("Should not have debug flag")
+	}
+}
+
+func TestColor(t *testing.T) {
+	if !IsColor(context.Background()) {
+		t.Errorf("Vanialla ctx should have color")
```

Comment:
typo

Reply:
Haha, thanks! That was intentional to check if you read everything :smirk: (not really)